src/ansys/aedt/core/visualization/advanced/hdm_plot.py

```diff
@@ -29,10 +29,25 @@
 
 from ansys.aedt.core.generic.constants import AEDT_UNITS
 from ansys.aedt.core.generic.general_methods import pyaedt_function_handler
-from ansys.aedt.core.generic.plot import CommonPlotter
-from ansys.aedt.core.generic.plot import ObjClass
-import numpy as np
-import pyvista as pv
+from ansys.aedt.core.visualization.plot.pyvista import CommonPlotter
+from ansys.aedt.core.visualization.plot.pyvista import ObjClass
```

Comment:
Seems like this file is not tested either.

Reply:
It is not tested directly, I agree, but we are testing indirectly from plot_field_from_fieldplot, which returns this object. 

In the future, I agree, we should create more atomic unit test. Probably this refactoring will help us to identify "post" problems. But maybe the unit tests should be refactored in a new PR. We can discuss.